layers/+email/mu4e/packages.el

```diff
@@ -26,32 +27,10 @@
       (global-set-key (kbd "C-x m") 'mu4e-compose-new))
     :config
     (progn
-      (evilified-state-evilify-map mu4e-main-mode-map
-        :mode mu4e-main-mode
-        :bindings
-        (kbd "j") 'mu4e~headers-jump-to-maildir)
-
-      (evilified-state-evilify-map
-       mu4e-headers-mode-map
-       :mode mu4e-headers-mode
-       :bindings
-       (kbd "C-j") 'mu4e-headers-next
-       (kbd "C-k") 'mu4e-headers-prev
-       (kbd "J") (lambda ()
-                   (interactive)
-                   (mu4e-headers-mark-thread nil '(read))))
```

Comment:
I would be sad to see the `J` binding go from the mu4e-header-mode and mu4e-view-mode. It's my primary way of processing mail. Basically they jump to the next thread marking the previous thread as read.